source/blog/2017-08-16-ovirt-ansible-roles-how-to-use.html.md

```diff
@@ -0,0 +1,149 @@
+---
+title: How to use oVirt Ansible roles
+author: omachace
+tags: Ansible, Roles, Automation, Configuration management
+date: 2017-08-16 15:00:00 CET
+comments: true
+published: true
+---
+
+In blog post [An Introduction to oVirt Ansible Roles](./2017-07-19-ovirt-ansible-roles-an-introduction.html.md) you can have read about new roles introduced in latest oVirt release. This blog post is follow up to show you how to set it up and use it.
+
+## Ansible Galaxy
+
+You may be familiar with [Ansible galaxy](https://galaxy.ansible.com/). Ansible galaxy stores many different Ansible roles, which may simplify your life. We store [oVirt Ansible roles](https://galaxy.ansible.com/oVirt/ovirt-ansible-roles/) in Ansible galaxy as well. In order to use oVirt roles from galaxy you need to perform following steps:
```

Comment:
"To use oVirt roles from Galaxy" -> "To install those roles from Ansible Galaxy"